src/main/java/com/github/vkorobkov/jfixturescmd/CmdArgs.java

```diff
@@ -0,0 +1,57 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.github.vkorobkov.jfixtures.JFixtures;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResultImpl;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.lang.reflect.Method;
+
+@Slf4j
+final class CmdArgs {
```

Comment:
Let's break `CmdArgs` into few classes - first is a plain stupid POJO with command line parameters definition and the second will do the logic?